deployments/helm/vl3/templates/vl3-nse-ucnf.tpl

```diff
@@ -151,3 +154,10 @@ spec:
       port: {{ .Values.metricsPort }}
       targetPort: monitoring
       protocol: TCP
+---
```

Comment:
The best practice is to add things in separate template files.  I know we don't always follow that.  

Also, I'm not sure if a role-binding is explicitly required for a service-account.... maybe not since in this case it will just be used authentication.  Not sure if you've researched that.

Reply:
It is not required in this case. Btw do you want me to split the file?